docs/resources/ui/widgets/built-in-widgets/list-grid.md

```diff
@@ -230,7 +232,20 @@ GridView provides a two-dimensional array of children. It is the widget of choic
 
 Like ListView, GridView only renders the visible items, making it efficient for displaying large collections of elements. GridView supports multiple configurations for column count, spacing, aspect ratio, and scroll directions, offering robust customization options for diverse layout needs.
 
+:::info
+GridView and StaggeredView are almost similar widgets in FlutterFlow, with the primary difference being that StaggeredView is typically used to create a layout similar to the Pinterest app and has fewer properties than GridView.
```

Comment:
Pinterest example should be added as an example at the end and not as part of the description as people may not know Pinterest app that well. You should talk about how StaggeredView is different from Grid View in terms of fixed vs unfixed size of children, and any other major differences. And then add an additional point about the Pinterest app example.

Reply:
Good point! Updated.